Replace Chalk with Picocolors

kytta opened the pull request:
[Colorette][1] is a very fast and lightweight color formatting library. It is faster than alternatives in both loading and calling times and saves precious space in `node_modules`.

Replacing Chalk here and in other oclif repos will sufficiently shrink the install size of the packages and declutter the dependency tree (Chalk ships five extra packages that aren't really needed for such a simple cause)

P. S. I have opened https://github.com/oclif/cli-ux/pull/417 before this PR, but then I replaced Nano Colors with Colorette in order not to take part in a [weird situation around the creation of the former][2]

[1]: https://github.com/jorgebucaran/colorette
[2]: https://twitter.com/jorgebucaran/status/1441713235980275718

kytta committed:
fix: replace Chalk with Nano Colors

Nano Colors [1] is a very fast and lightweight color formatting library. It is
faster than alternatives in both loading and calling times and saves precious
space in `node_modules`.

Replacing Chalk here and in other oclif repos will sufficiently shrink the
install size of the packages and declutter the dependency tree (Chalk ships
five extra packages that aren't really needed for such a simple cause)

[1]: https://github.com/ai/nanocolors

kytta committed:
Replace Nano Colors with Colorette

yordis commented:
What about https://github.com/alexeyraspopov/picocolors ?

NickKaramoff commented:
> What about [alexeyraspopov/picocolors](https://github.com/alexeyraspopov/picocolors)? ([alexeyraspopov/picocolors#45](https://github.com/alexeyraspopov/picocolors/issues/45))

Yep, that would be my package of choice. It didn't exist back when I opened this PR. I'll add new commit later 👍

kytta committed:
Replace `colorette` with `picocolors`

`picocolors` is a more minimalistic and a faster library than
`colorette`. It is used by many big libraries, like PostCSS, Stylelint,
Browserslist, etc.

kytta committed:
Remove dependency on `supports-color`

kytta committed:
Merge branch 'master' into colorette
